packages/smooth_app/lib/database/dao_product_extra.dart

```diff
@@ -3,6 +3,7 @@ import 'dart:collection';
 import 'dart:convert';
 
 import 'package:diacritic/diacritic.dart';
+import 'package:flutter/cupertino.dart';
```

Comment:
Doesn't look like it's needed here

Reply:
I was surprised, too, when I reviewed my own code.
`cupertino.dart` is needed for `debugPrint` later in the code.
Is `debugPrint` actually needed, probably not, but that's the best I can do for the moment as I cannot reproduce the bug myself.
And thank you for the approval!